plugin/plugin.py

```diff
@@ -39,7 +39,7 @@ def init(self, spec_bytes, no_connection: bool = False):
 
     def get_tables(self, options: plugin.TableOptions) -> List[plugin.Table]:
         all_tables: List[plugin.Table] = [
-            tables.Bitlinks(self._spec.extract_utm),
+            tables.Bitlinks(self._spec.extract_utm, self._spec.countries_summary_unit),
```

Comment:
Don't we need to list all of the tables here? or are the others set up as strict children?

Reply:
I don't think you can sync a child table only, or can you?

Reply:
It's consistent with `Square` so I presume this implementation is correct.